src/test/java/games/strategy/engine/framework/map/download/MapDownloadControllerTest.java

```diff
@@ -42,17 +42,8 @@ public void testShouldPromptToDownloadTutorialMap_ShouldReturnFalseWhenCanPrompt
   }
 
   @Test
-  public void testShouldPromptToDownloadTutorialMap_ShouldReturnFalseWhenCannotPromptToDownloadAndUserMapsIsEmpty() {
+  public void testShouldPromptToDownloadTutorialMap_ShouldReturnFalseWhenCannotPromptToDownload() {
     when(tutorialMapPreferences.canPromptToDownload()).thenReturn(false);
-    when(userMaps.isEmpty()).thenReturn(true);
-
-    assertThat(shouldPromptToDownloadTutorialMap(), is(false));
-  }
-
-  @Test
-  public void testShouldPromptToDownloadTutorialMap_ShouldReturnFalseWhenCannotPromptToDownloadAndUserMapsIsNotEmpty() {
-    when(tutorialMapPreferences.canPromptToDownload()).thenReturn(false);
-    when(userMaps.isEmpty()).thenReturn(false);
```

Comment:
`userMaps.isEmpty()` is never called for these two tests.  Therefore, the tests are equivalent.  So I removed one and renamed the other to indicate the state of `userMaps` is no longer considered.